shared/actions/kbfs/index.js

```diff
@@ -1,39 +1,3 @@
 // @flow
-import * as KBFSGen from '../kbfs-gen'
-import * as Platform from './index.platform'
-import * as Saga from '../../util/saga'
-import * as RPCTypes from '../../constants/types/flow-types'
-import {isWindows} from '../../constants/platform'
-
-function* _listSaga(action: KBFSGen.ListPayload): Saga.SagaGenerator<any, any> {
-  try {
-    const result: ?RPCTypes.ListResult = yield Saga.call(RPCTypes.fsListRpcPromise, {
-      path: action.payload.path,
-    })
-
-    if (result) {
-      console.log('fs.List: ', result)
-      yield Saga.put(KBFSGen.createListed({result}))
-    }
-  } catch (error) {
-    yield Saga.put(KBFSGen.createListedError({error}))
-  }
-}
-
-function* kbfsSaga(): Saga.SagaGenerator<any, any> {
-  yield Saga.safeTakeLatest(KBFSGen.list, _listSaga)
-  yield Saga.safeTakeEvery(KBFSGen.open, Platform.openSaga)
-  yield Saga.safeTakeEvery(KBFSGen.openInFileUI, Platform.openInFileUISaga)
-  yield Saga.safeTakeLatest(KBFSGen.fuseStatus, Platform.fuseStatusSaga)
-  yield Saga.safeTakeLatest(KBFSGen.fuseStatusUpdate, Platform.fuseStatusUpdateSaga)
-  yield Saga.safeTakeLatest(KBFSGen.installFuse, Platform.installFuseSaga)
```

Comment:
i moved this one line while moving it into platform. we take and run installfuseSaga, but then also have an if that does something different on windows. i assume this is incorrect and we want one or the other